src/common/types.ts

```diff
@@ -598,6 +598,18 @@ export interface GamesDBInfo {
   steamID: string
 }
 
+export interface ProtonCompatibility {
```

Comment:
Better call it `ProtonDBInfo`

Reply:
ProtonDBInfo is the JSON that the API returns, should we keep it all? I don't think most is relevant.

Reply:
I just want to rename it more generic because maybe later it does contain information about compatibility if protondb decide to extend there api.